apps/session-manager/package.json

```diff
@@ -0,0 +1,44 @@
+{
+    "name": "@pagopa/io-session-manager",
+    "version": "0.0.1",
+    "description": "API for IO session",
+    "main": "./dist/main.js",
+    "license": "MIT",
+    "scripts": {
+        "clean": "shx rm -rf dist",
+        "clean:generated": "shx rm -rf ./src/generated",
+        "build": "tsc",
+        "build:watch": "yarn build -w",
+        "typecheck": "yarn build --noemit",
+        "start": "node dist/server.js",
+        "start:watch": "yarn start watch",
+        "lint": "yarn lint:check --fix",
+        "lint:check": "eslint . -c .eslintrc.js --ext .ts,.tsx --cache",
+        "test": "vitest run",
+        "test:watch": "vitest"
+    },
+    "devDependencies": {
+        "@pagopa/typescript-config-node": "*",
+        "@types/express": "^4.17.21",
+        "@types/node": "~20.10.0",
+        "dependency-check": "^4.1.0",
+        "dotenv-cli": "^7.4.1",
+        "eslint": "^8.57.0",
+        "eslint-plugin-prettier": "^5.1.3",
+        "prettier": "^3.2.5",
+        "shx": "^0.3.2",
+        "swagger-cli": "^4.0.4",
+        "typescript": "^5.4.3"
+    },
+    "dependencies": {
+        "@azure/functions": "^3.2.0",
+        "@pagopa/io-functions-commons": "^28.21.0",
+        "@pagopa/io-spid-commons": "^13.4.0",
+        "@pagopa/openapi-codegen-ts": "^13.1.0",
+        "@pagopa/ts-commons": "^12.6.0",
+        "express": "^4.19.2",
+        "fp-ts": "^2.16.5",
+        "io-ts": "^2.2.21",
+        "vitest": "^1.5.0"
```

Comment:
should be a `devDependency`

Reply:
Right, thanks! Moved in devDependency in [[#IOPID-1763] add eslint + fix based on code review](https://github.com/pagopa/io-auth-n-identity-domain/pull/1/commits/95787ff45a3ddbfeccbb0f616623810c957e883f)